docker/core/Makefile

```diff
@@ -0,0 +1,59 @@
+.RECIPEPREFIX +=
```

Comment:
This file is a copy of another makefile in the repo – I don't think all of that is needed here. Let's only leave the stuff that this container supports (e.g. it definitely doesn't support `serve`)

Reply:
Yeah, nice comment. Fill fix it.